src/databricks/sql/utils.py

```diff
@@ -595,6 +653,26 @@ def _convert_column_to_arrow_array(t_col):
     raise OperationalError("Empty TColumn instance {}".format(t_col))
 
 
+def _covert_column_to_list(t_col):
+    supported_field_types = (
```

Comment:
can we define it as the static or constant variable if this is constant.

Reply:
The function is not part of a class, so I don't think we can make it static. In Python I believe we can't make anything constant, so I have replaced the names with ALL CAPS to indicate that this is a constant